js/build.js

```diff
@@ -9,6 +9,8 @@ Fliplet.Widget.instance('notification-inbox-1-0-0', function(data) {
   // Sample implementation to initialize the widget
   var inbox = new NotificationInbox(element, data);
 
+  $(element).translate();
```

Comment:
Does this component use any translation strings from translation.json? If not, then this line is unnecessary.

```suggestion
  $(element).translate();
```

Reply:
@tonytlwu Isn't it necessary for TD and TN helpers in templanes?

Reply:
@OleksandrLavrynenko No, it isn't. `$(selector).translate()` is only required to translate the HTML elements with `data-translate` attributes.

Reply:
Ok, I'll fix it